x/concentrated-liquidity/pool.go

```diff
@@ -100,7 +100,7 @@ func (k Keeper) CalcOutAmtGivenIn(ctx sdk.Context,
 	tokenOutDenom string,
 	swapFee sdk.Dec,
 	priceLimit sdk.Dec,
-	poolId uint64) (tokenIn, tokenOut sdk.Coin, updatedTick sdk.Int, updatedLiquidity sdk.Dec, err error) {
+	poolId uint64) (tokenInDelta, tokenOutDelta sdk.Int, updatedTick sdk.Int, updatedLiquidity sdk.Dec, err error) {
```

Comment:
Im wondering if we should do tokenIn and tokenOut instead? ( I personally think this way would be more useful as API) since the caller would have to recalculate on the actual token In and token Out. What do you think?

Reply:
These terms are actually sysnonymous, since the delta is what we are putting into and taking out of the pool. But agreed that tokenIn and tokenOut are just simpler to understand / better API

Reply:
Changed!